VideoApi/VideoApi.UnitTests/Services/VirtualRoom/GetOrCreateAnInterpreterVirtualRoomTests.cs

```diff
@@ -121,6 +121,7 @@ public async Task should_create_vmr_with_kinly_if_room_is_not_available()
             room.Label.Should().Be(newVmrRoom.Room_label);
             room.PexipNode.Should().Be(newVmrRoom.Uris.Pexip_node);
             room.ParticipantUri.Should().Be(newVmrRoom.Uris.Participant);
+            room.IngestUrl.Contains("_interpreter_").Should().Be(true);
```

Comment:
I feel this can be more specific. It should be the same as on line 111 right? Could probably have an "expectedIngestUrl" set at the start of the method.